src/features/account/update-basic-info-drawer/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {useForm} from 'react-hook-form';
+import {useEffect} from 'react';
+import FormInput from '@/features/shared/components/form-input';
+import 'react-calendar/dist/Calendar.css';
```

Comment:
Is this import necessary? Also, for Calendars let use the ones MUI provide, they are great.